nostr/relay.py

```diff
@@ -60,8 +65,27 @@ def connect(self):
     def close(self):
         self.ws.close()
 
+    def check_reconnect(self):
+        try:
+            self.close()
+        except:
+            pass
+        self.connected = False
+        if self.reconnect:
```

Comment:
Where does `self.reconnect` change? I only see it getting set to `True` in `__post_init__`.

Is it actually needed?

Reply:
It was meant as a configuration in the class init so the "user" can choose whether they want reconnect attempts or not.